.github/workflows/canary-integration-test.yml

```diff
@@ -34,12 +34,17 @@ jobs:
         kubectl cluster-info
         kubectl get pods -n kube-system
 
-    - name: use local disk
+    - name: use local disk and create partitions for osds
       run: |
+        BLOCK=$(sudo lsblk --paths|awk '/14G/ {print $1}'| head -1)
```

Comment:
To reduce duplication between all these tests, can we factor out this whole section into a script that all the actions will call?

Reply:
https://github.com/rook/rook/issues/7376